packages/components/src/components/primitives/Input/Input.test.tsx

```diff
@@ -0,0 +1,245 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { renderWithProviders } from 'tests/renderWithProviders';
+import { fireEvent, screen } from '@testing-library/react';
+import { Input } from '.';
+import { ComposerProvider } from 'theme';
+import { useMultiStyleConfig } from '@chakra-ui/react';
+import {
+  flushedStyle,
+  getIconStyle,
+  getInputGroupStyle,
+  getLabelStyle,
+  iconStyles,
+  inputGroupStyles,
+  labelStyles,
+} from 'theme/ComposerTheme/Components/Input';
+
+jest.mock('@chakra-ui/react', () => ({
+  ...(jest.requireActual('@chakra-ui/react') as any),
+  useMultiStyleConfig: jest.fn().mockReturnValue({
+    element: {},
+    elementContainer: {},
+    inputGroup: {},
+    field: { withIcon: {} },
+    leftLabel: {},
+    rightLabel: {},
+  }),
+}));
+
+const mockFlushedStyle = {
+  inputGroup: {
+    borderRadius: 0,
+    ...getInputGroupStyle({}, 'flushed'),
+  },
+  leftLabel: {
+    ...labelStyles,
+    ...getLabelStyle({}, 'flushed'),
+  },
+  rightLabel: {
+    ...labelStyles,
+    ...getLabelStyle({}, 'flushed'),
+  },
+  element: {
+    color: 'text-secondary',
+    ...getIconStyle({}),
+  },
+  elementContainer: {
+    height: '100%',
+  },
+  field: {
+    color: 'text-primary',
+    backgroundColor: 'background-action-default',
+    border: 'none',
+    borderColor: 'ui-element-outline-default',
+    borderRadius: 0,
+    _placeholder: {
+      color: 'text-secondary',
+      textStyle: 'text-body-regular',
+    },
+    _hover: {
+      backgroundColor: 'background-action-hover',
+      border: 'none',
+      borderBottom:
+        '0.063rem solid var(--chakra-colors-ui-element-outline-active)',
+    },
+    _focus: {
+      boxShadow: 'none',
+      border: 'none',
+      borderBottom:
+        'solid 0.063rem var(--chakra-colors-ui-element-outline-default)',
+      borderColor: 'ui-element-outline-default',
+    },
+    _invalid: {
+      boxShadow: 'none',
+      border: 'none',
+      borderColor: 'transparent',
+    },
+    _disabled: {
+      _placeholder: {
+        color: 'text-disabled',
+      },
+      color: 'text-secondary',
+      backgroundColor: 'background-action-disabled',
+      border: 'solid 0.063rem var(--chakra-colors-ui-element-outline-disabled)',
+    },
+  },
+};
```

Comment:
IMO we should not be testing styling via unit tests, we already have chromatic for this type of verifications

Reply:
Added this test because of Codecov asking me to add testing coverage to the `flushedStyle` theme function. Should I remove it?

Reply:
if already implemented I don't see a good reason to remove it 😉, IMO in general I will not expend time in unit testing things that are already verified by chromatic